blogs/2016/04/22/edge-debugging-for-vs-code.md

```diff
@@ -0,0 +1,48 @@
+---
+Order: 2
+TOCTitle: Microsoft Edge Debugging for VS Code
+PageTitle: Microsoft Edge Debugging for VS Code
+MetaDescription: Microsoft Edge Debugging for Visual Studio Code
+Date: 2016-04-22
+ShortDescription: As a part of our continuous effort to simplify the daily workflow for developers, we are happy to enable web developers to debug their code running inside Microsoft Edge, directly from the editor by introducing Microsoft Edge Debugging for Visual Studio Code.
+
+Author: Kenneth Auchenberg
+---
+
+# Introducing Chrome Debugging for VS Code
+
+April 22, 2016 by [Andy Sterland](https://twitter.com/AndySterland) and [Kenneth Auchenberg](https://twitter.com/auchenberg)
+
+As a part of our continuous effort to simplify the daily workflow for developers, we are happy to enable web developers to debug their code running inside Microsoft Edge, directly from the editor by introducing Microsoft Edge Debugging for Visual Studio Code.
+
+Our new Edge Debugger, works is quite similar to [our Chrome debugger](https://code.visualstudio.com/blogs/2016/02/23/introducing-chrome-debugger-for-vs-code) which we introduced back in February, and technically it’s the same debugger running inside VS Code, as we have embraced the [Chrome Debugger protocol](https://developer.chrome.com/devtools/docs/debugger-protocol) with our new Edge Diagnostics Adaptor, which we have written in detail about on the [Edge blog]().
```

Comment:
Our new Edge Debugger works similar to our ... which we introduced back in February.  Technically, its the same debugger running inside VS Code as we used the ... with our new Edge Diagnostics Adapter.  You can learn the details in our [Edge blog].